src/ExpressionEvaluator/Core/Test/ResultProvider/Debugger/Engine/DkmWorkList.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+#region Assembly Microsoft.VisualStudio.Debugger.Engine, Version=1.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
+// D:\Roslyn\Main\Open\Binaries\Debug\Microsoft.VisualStudio.Debugger.Engine.dll
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace Microsoft.VisualStudio.Debugger
+{
+    public delegate void DkmWorkListCompletionRoutine(DkmWorkList workList);
+
+    public sealed class DkmWorkList : MarshalByRefObject
```

Comment:
Can we simplify this? Perhaps:
class DkmWorkList
{
    void Execute() { ... }
    void AddWork(Action) { ... }
}